mopidy/core/playlists.py

```diff
@@ -16,24 +17,65 @@ def __init__(self, backends, core):
         self.backends = backends
         self.core = core
 
-    """
-    Get the available playlists.
+    def as_list(self):
+        """
+        Get a list of the currently available playlists.
 
-    Returns a list of :class:`mopidy.models.Playlist`.
-    """
-    def get_playlists(self, include_tracks=True):
-        futures = [b.playlists.playlists
-                   for b in self.backends.with_playlists.values()]
+        Returns a list of :class:`~mopidy.models.Ref` objects referring to the
+        playlists. In other words, no information about the playlists' content
+        is given.
+
+        :rtype: list of :class:`mopidy.models.Ref`
+
+        .. versionadded:: 1.0
+        """
+        futures = [
+            b.playlists.as_list()
+            for b in self.backends.with_playlists.values()]
         results = pykka.get_all(futures)
```

Comment:
Should we bother with catching the potential not implemented exceptions from this in core?